mmpose/models/backbones/tcn.py

```diff
@@ -0,0 +1,197 @@
+import copy
+
+import torch.nn as nn
+from mmcv.cnn import ConvModule, build_conv_layer
+
+from ..registry import BACKBONES
+from .base_backbone import BaseBackbone
+
+
+class BasicTemporalBlock(nn.Module):
+    """Basic block for VideoPose3D (simple yet effective baseline).
```

Comment:
> (simple yet effective baseline)

Is this a private comment or a paper title

Reply:
A paper title

Reply:
interesting

Reply:
The paper title is "3D human pose estimation in video with temporal convolutions and semi-supervised training", isn't it?

Reply:
Yes. And this paper can be regarded as an expanded version of another paper -- " A simple yet effective baseline for 3d human pose estimation". Their architectures are similar.

Reply:
Do you want to make these two models to share `BasicTemporalBlock`?
I think that's a little bit confusing. Because "A simple yet effective baseline for 3d human pose estimation" focus on **image-based** 3d pose estimation, instead of VideoPose3D. 

Better add another backbone file for "A simple yet effective baseline for 3d human pose estimation".

Reply:
OK, I'll modify the docstring. So this backbone is only for temporal input.